modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]AllowUsersToDefineAPIApplicationEndpoints/EditAPIApplicationEndpoint.testcase

```diff
@@ -35,6 +35,67 @@ definition {
 		}
 	}
 
+	@priority = 3
+	test CanChangeSchemaOfEndpoint {
+		property portal.acceptance = "true";
+
+		task ("Given published API Application with Endpoint created") {
+			EndpointAPI.createAPIEndpoint(
+				apiApplicationERC = "My-app",
+				name = "testendpoint1",
+				path = "/testendpoint1");
+		}
+
+		task ("two API Schema in API Application") {
```

Comment:
In headless qa team, the GWT steps should not be updated. We would keep here
`task("And Given two API Schema in API Application"){}`